shared/profile/pgp/finished-generating-pgp.desktop.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import React, {Component} from 'react'
+import {StandardScreen, Box, Text, Button, Icon, Checkbox} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../styles/style-guide'
+import type {Props} from './finished-generating-pgp'
+
+type State = {
+  shouldStoreKeyOnServer: boolean,
+}
+
+class FinishedGeneratedPgp extends Component<void, Props, State> {
+  state: State;
+
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      shouldStoreKeyOnServer: true,
+    }
+  }
+
+  _onCheckToggle (shouldStoreKeyOnServer: boolean) {
+    this.setState({shouldStoreKeyOnServer})
+  }
+
+  render () {
+    return (
+      <StandardScreen notification={{type: 'success', message: 'Your PGP key was generated!'}}>
+        <Icon style={styleIcon} type='icon-pgp-key-48' />
+        <Text style={styleTitle} type='Header'>Here is your unique public key!</Text>
+        <textInput style={stylePgpKeyString} readOnly={true}>{this.props.pgpKeyString}</textInput>
+        <Box style={styleUploadContainer}>
+          <Checkbox onCheck={(newVal) => this._onCheckToggle(newVal)} checked={this.state.shouldStoreKeyOnServer} />
```

Comment:
why not have this first line just be the checkbox and just 1 Text below. This way if you click on the text it'll check the box like you'd expect